packages/perseus/src/widgets/interactive-graph.tsx

```diff
@@ -2351,7 +2352,8 @@ class InteractiveGraph extends React.Component<Props, State> {
             throw makeInvalidTypeError("getAngleEquationString", "angle");
         }
         const coords = InteractiveGraph.getAngleCoords(props.graph, props);
-        const angle = GraphUtils.findAngle(coords[2], coords[0], coords[1]);
+        const allowReflexAngles = props.graph.allowReflexAngles || false;
+        const angle = getClockwiseAngle(coords, allowReflexAngles);
         return (
```

Comment:
These options needed to be added in order to update the graph previews while editing these options in the Content Editor.